src/pages/steps/LocationStep.tsx

```diff
@@ -224,7 +224,7 @@ const LocationStep = ({
   return (
     <Stack
       {...getStackProps(props)}
-      minHeight={isNameAndAgeRangeVisible ? 'inherit' : '350px'}
+      minHeight={isNameAndAgeRangeVisible ? 'inherit' : '400px'}
```

Comment:
```suggestion
      minHeight={isNameAndAgeRangeVisible ? 'inherit' : '26.5rem'}
```

To make it more adaptable to possible font changes in the future.